jsonnet/kube-thanos/kube-thanos-receive-hashrings.libsonnet

```diff
@@ -1,5 +1,5 @@
-local receiveConfigDefaults = import 'kube-thanos/kube-thanos-receive-default-params.libsonnet';
-local receive = import 'kube-thanos/kube-thanos-receive.libsonnet';
+local receiveConfigDefaults = import './kube-thanos-receive-default-params.libsonnet';
```

Comment:
Or you can use full import path of `github.com/thanos-io/thanos/kube-thanos/kube-thanos-receive-default-params.libsonnet` which is guaranteed to be correct and allows to ditch legacy imports in jsonnet-bundler.

Reply:
That is how I use the library, but why would you do that in the library?
And it is still not guaranteed to work, for example here how it is installed in the repo itself:
https://github.com/thanos-io/kube-thanos/blob/e1a68590f56034ca1a43d59401f03e72fd01ac5f/jsonnetfile.json#L4-L11
It create `vendor/kube-thanos`. It it not guarantee either that the user will use `jb`, especially that this library does not have any dependency. Another problematic case is if the user wants to use a fork, they need to rename those imports before being able to use it.